packages/conf/src/types.ts

```diff
@@ -37,7 +37,7 @@ export type CatalogFormatOptions = {
   disableSelectWarning?: boolean
 }
 
-export type OrderBy = "messageId" | "origin"
+export type OrderBy = "messageId" | "message" | "origin"
```

Comment:
I thought about renaming or adding `linguiId` as an option, but feel like it may be misleading itself, as we do use custom message id as messageId if exists. And support purely order by `linguiId` seems not that helpful at the moment. So only adding the `message` here.